API/SSW.Rewards.Persistence/Migrations/20201014040152_AddPostalAddress.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace SSW.Rewards.Persistence.Migrations
+{
+    public partial class AddPostalAddress : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<int>(
```

Comment:
Could we collapse these migrations into one for prod?

Reply:
We could. I didn't feel like it was a good use of time for this PR. If the idea is to ready it for a prod deployment, ideally we would collapse all migrations into one, in which case this would be an isolated piece of work anyway.